client/src/entrypoints/admin/core.js

```diff
@@ -15,19 +15,6 @@ function addMessage(status, text) {
 
 window.addMessage = addMessage;
 
-function escapeHtml(text) {
-  const map = {
-    '&': '&amp;',
-    '<': '&lt;',
-    '>': '&gt;',
-    '"': '&quot;',
-    "'": '&#039;',
-  };
-
-  return text.replace(/[&<>"']/g, (char) => map[char]);
-}
-
-window.escapeHtml = escapeHtml;
```

Comment:
Removing this `window.escapeHtml = escapeHtml;` line will mean that the function is no longer available globally to all Javascript - only to files that explicitly import it from utils. Right now it looks like add-multiple.js [in wagtailimages](https://github.com/wagtail/wagtail/blob/05951b350196b059307d4c0233fdf773d4b41486/wagtail/images/static_src/wagtailimages/js/add-multiple.js#L38) and [wagtaildocs](https://github.com/wagtail/wagtail/blob/05951b350196b059307d4c0233fdf773d4b41486/wagtail/documents/static_src/wagtaildocs/js/add-multiple.js#L27) relies on the global definition.

We should either have core.js import the definition of `escapeHtml` from utils/text.js, and then make it available globally by keeping this line in place - or, preferably, move the two add-multiple.js files into the appropriate places in client/src/entrypoints and update them to explicitly import `escapeHtml` from utils. (That might be out of scope for this issue, though.)

Reply:
okay noted thanks let me make the changes

Reply:
Yes. As per the issue checklist - we need to ensure the existing code can still access the global. 

Or, as per above suggestion - we can import the util via moving the other files that use it. Although that may be a harder - but feel free to give it a go.